static/js/apps/nl_interface/query_welcome.tsx

```diff
@@ -58,11 +47,13 @@ export function QueryWelcome(props: QueryWelcomePropType): JSX.Element {
     }
     // If a topic is set, show sample queries from that topic
     // Otherwise show a random sample from all topics
-    const topicQueries = _.groupBy<SampleQuery>(allQueries, "topic");
-    if (topic.current && topicQueries[topic.current]) {
-      setExampleQueries(
-        topicQueries[topic.current].filter((q) => q.wai).slice(0, 12)
-      );
+    const topicQueries = _.groupBy<SampleQuery>(
+      allQueries,
+      "topic"
+    ) as TopicQueries;
+    setTopicQueries(topicQueries);
+    if (topic && topicQueries[topic]) {
+      setExampleQueries(topicQueries[topic].filter((q) => q.wai).slice(0, 12));
```

Comment:
nit: make 12 a constant along the lines of MAX_EXAMPLE_QUERIES

Reply:
Done!